webhooks/ssp_webhook.go

```diff
@@ -32,17 +33,27 @@ import (
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 
+	sspv1beta1 "kubevirt.io/ssp-operator/api/v1beta1"
 	sspv1beta2 "kubevirt.io/ssp-operator/api/v1beta2"
 	sspv1beta3 "kubevirt.io/ssp-operator/api/v1beta3"
 )
 
+// +kubebuilder:webhook:verbs=create;update,path=/validate-ssp-kubevirt-io-v1beta1-ssp,mutating=false,failurePolicy=fail,groups=ssp.kubevirt.io,resources=ssps,versions=v1beta1,name=validation.v1beta1.ssp.kubevirt.io,admissionReviewVersions=v1,sideEffects=None
 // +kubebuilder:webhook:verbs=create;update,path=/validate-ssp-kubevirt-io-v1beta2-ssp,mutating=false,failurePolicy=fail,groups=ssp.kubevirt.io,resources=ssps,versions=v1beta2,name=validation.v1beta2.ssp.kubevirt.io,admissionReviewVersions=v1,sideEffects=None
 // +kubebuilder:webhook:verbs=create;update,path=/validate-ssp-kubevirt-io-v1beta3-ssp,mutating=false,failurePolicy=fail,groups=ssp.kubevirt.io,resources=ssps,versions=v1beta3,name=validation.v1beta3.ssp.kubevirt.io,admissionReviewVersions=v1,sideEffects=None
 
 var ssplog = logf.Log.WithName("ssp-resource")
 
 func Setup(mgr ctrl.Manager) error {
 	err := ctrl.NewWebhookManagedBy(mgr).
+		For(&sspv1beta1.SSP{}).
+		WithValidator(newSspValidator(mgr.GetClient())).
+		Complete()
+	if err != nil {
+		return fmt.Errorf("failed to create webhook for v1beta2.SSP")
```

Comment:
```suggestion
		return fmt.Errorf("failed to create webhook for v1beta1.SSP")
```

But should it actually create a webhook, since it is not served anymore?

Reply:
I've removed the webhook code. It will not be called, because the API is not served.